src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Should not `import java.util.*`, introduces ambiguity and could introduce possible naming conflicts in the future.
For example:

`java.lang.reflect.Array`
`java.sql.Array`
So, if you import `java.lang.reflect.*` and `java.sql.*` you'll have a collision on the Array type, and have to fully qualify them in your code.

Importing specific classes instead will save you this hassle.
Source: https://stackoverflow.com/questions/187453/import-package-vs-import-package-specifictype